src/Plugin/QueueItem/FileItem.php

```diff
@@ -25,6 +25,13 @@ public function __construct(array $data = []) {
     $this->file = $data['file'];
     $this->url = $data['url'] ?? NULL;
     $this->fullPath = $data['full_path'] ?? NULL;
+
+    // Handle responsive images by removing extraneous data like ` 1x`.
+    // @todo Fix this in the Quant infrastructure.
+    if ($this->fullPath) {
```

Comment:
This code looks fine, but I'm not sure why were transforming path on disk — do image styles not include the ` 1x` when resaving?

Reply:
@steveworley We're not changing the path on disk afaik. In some situations (unknown why, assume a core bug), when you get the "full_path", it shows a space and then ` 1x` at the end even though that is not correct and should not be there.

Stu said he will handle this at some point on the infra side, but this is just a workaround for now.

Without this workaround, then some of the image styles may not be created if they don't exist already.